README.md

```diff
@@ -1,6 +1,58 @@
 # covid19-india
 
+## Repo Structure
+    The repository is structured as follows:
+    ./
+    |-- config    : Various config params to be entered by humans go here
+    |-- data      : All data go in here, files are tracked by git-lfs
+    |-- outputs   : All output data go in here, files are tracked by git-lfs
+    |-- docs      : All documentation other than README goes here
+    |-- Makefile  : Makefile to clean data, run tests etc
+    |-- notebooks : All notebooks go here
+    |-- setup.sh  : Setup script to setup path etc
+    `-- src       : All production code, resusable code components go here
+        |-- configs        : All configs params go here
+        |-- data_fetchers  : These are utilities for fetching data etc
+        |-- driver.py      : Sample Code
+        |-- entities       : Important entities, these are provided for data verification
+        |-- model_wrappers : wraps main models, currently seir.py wraps the seirsplus models
+            |-- base.py                            : has Abstract Base Class for wrapping models
+            |-- model_factory.py                   : has model factory to build models
+            |-- seir.py                            : SEIR model wrapper, which wraps around seirsplus model
+            |-- intervention_enabled_model_base.py : General Wrapper for allowing interventions
+            `-- intervention_enabled_seir.py       : Wrapper for seir model, only just maps variables
+        |-- modules        : 
+            |-- forecasting_module.py              : Makes prediction for a given region (and region type) for start-end date.
+            |-- model_evaluator.py                 : Various evaluation utils, for_region, for various loss funcs etc
+            |-- training_module.py                 : Training module, this currently used hyperopt to optimize
+            |-- data_fetcher_module.py             : Data fetcher module for fetching various data into requisite format
+            `-- scenario_forecasting_module.py     : Scenario based forecasting module.
+        `-- utils          : Various tools like loss_funcs, hyperopt wrapper etc
 
+
+## Class Structure
+
+## Work-Flow
+1. We follow [git-flow](https://nvie.com/posts/a-successful-git-branching-model/) with staging in our parlance being develop in theirs.
+2. Ensure that your code satisfies [pep8](https://www.python.org/dev/peps/pep-0008/)
+3. Ensure that you have documentation as per [pep257](https://www.python.org/dev/peps/pep-0257/) 
+4. TODO: Decide PEP257, PEP287, something else.
```

Comment:
What do you mean  by todo decide ?

Reply:
It's task reminder for me to decide which docstring format we want to use.